main/index.js

```diff
@@ -75,18 +78,24 @@ const buildApp = (options, outDir, cleanup) => {
         }
         console.log(`Build of ${entry.file} successful`);
         if (!options.watch) {
-          cleanup().catch(console.error);
+          cleanup()
+            .catch(console.error)
+            .then(() => resolve(cleanup));
```

Comment:
why resolve with the cleanup function?

Reply:
Because I am handling SIGINT in `bin/build.js` which does the cleanup before exiting.